zeppelin-server/src/main/java/org/apache/zeppelin/service/ShiroAuthenticationService.java

```diff
@@ -138,8 +139,8 @@ private String extractPrincipal(Subject subject) {
   @Override
   public Collection<Realm> getRealmsList() {
     String key = ThreadContext.SECURITY_MANAGER_KEY;
-    DefaultWebSecurityManager defaultWebSecurityManager = (DefaultWebSecurityManager) ThreadContext.get(key);
-    return defaultWebSecurityManager.getRealms();
+    DefaultSecurityManager defaultSecurityManager = (DefaultSecurityManager) ThreadContext.get(key);
```

Comment:
Using a less specific class made it easier to test (hit a weird issue trying to create a DefaultWebSecurityManager and it seemed unnecessary)